Source/TCPWrapper/Public/TCPClientComponent.h

```diff
@@ -73,6 +70,9 @@ class TCPWRAPPER_API UTCPClientComponent : public UActorComponent
 	*/
 	UFUNCTION(BlueprintCallable, Category = "TCP Functions")
 	bool Emit(const TArray<uint8>& Bytes);
+	
+	UFUNCTION(BlueprintCallable, Category = "TCP Functions")
+	bool IsConnected();
```

Comment:
ideally there would be an event driven way to sync the connection state rather than a polled state, but this might be a clean enough fix for now